packages/calypso-polyfills/.eslintrc.js

```diff
@@ -2,4 +2,12 @@ module.exports = {
 	rules: {
 		'import/no-extraneous-dependencies': [ 'error', { packageDir: __dirname } ],
 	},
+	overrides: [
+		{
+			files: [ '*.md.js' ],
+			rules: {
+				'import/no-extraneous-dependencies': 'off',
+			},
+		},
+	],
```

Comment:
Maybe it was discussed somewhere, but shouldn't these overrides be defined at some global place, instead of per-package.

If a documentation example contains JS code for `import`, it seems outright absurd to me to check if the imported package is declared as `package.json` dependency.

Reply:
They are defined in a the global `.eslintrc.js` (https://github.com/Automattic/wp-calypso/pull/45802), but because this module has its own `.eslintrc.js` that re-enables `import/no-extraneous-dependencies` for all files, we have to disable for code blocks again.